core/src/test/java/org/apache/calcite/util/SourceTest.java

```diff
@@ -118,6 +120,22 @@ void testAbsoluteFileToUrl(String path, String expectedUrl) throws URISyntaxExce
         () -> "Sources.of(Sources.of(file(" + path + ").absolutePath).url()).file().getPath()");
   }
 
+  /**
+   * [CALCITE-5052] Resources created with the JAR protocol should also work.
```

Comment:
if youre going to link a jira case you can format it like this example: https://github.com/apache/calcite/blob/main/core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java#L277